src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -168,8 +168,9 @@ private class MinDateAndDateTimeAgg : IAggregator
             protected DateTime _minValueDT = DateTime.MaxValue;
             protected int _count = 0;
 
-            public void Apply(FormulaValue value)
+            public void Apply(FormulaValue value, EvalVisitor runner)
```

Comment:
>EvalVisitor runner [](http://example.com/codeflow?start=50&length=18)

Can we avoid passing EvalVisitor and instead pass a more restrictive object?  "Least privilege" principle - EV is too powerful 

Like runtimeConfig or IServiceProvider?

Reply:
Agree, Changing it to IServiceProvide